native/packages/react-native-bpk-component-icon/stories.js

```diff
@@ -42,13 +47,102 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     flexWrap: 'wrap',
   },
+  column: {
+    flexDirection: 'column',
+    marginBottom: spacingBase,
+  },
+  singleRow: {
+    minWidth: 100,
+  },
+  singleIcon: {
+    paddingLeft: spacingSm,
+    paddingRight: spacingSm,
+  },
   icon: {
-    color: colorBlue700,
+    color: colorBlue500,
+  },
+  bottomMargin: {
+    marginBottom: spacingSm,
   },
 });
 
+/*
+ * the following functions should go on the bottom
+ * but eslint complains about it -_-
+ * @return
+ */
+function getSmallIcons() {
+  return (<View style={styles.column}>
+    <Text style={styles.bottomMargin}>Small:</Text>
+    <View style={[styles.singleRow, styles.group]}>
+      <BpkIcon
+        style={styles.singleIcon}
+        icon="flight"
+        small
+      />
+      <BpkIcon
+        style={styles.singleIcon}
+        icon="cars"
+        small
+      />
+      <BpkIcon
+        style={styles.singleIcon}
+        icon="hotels"
+        small
+      />
+    </View>
+  </View>);
+}
+
+function getLargeIcons() {
+  return (<View style={styles.column} >
+    <Text style={styles.bottomMargin}>Large:</Text>
+    <View style={[styles.singleRow, styles.group]}>
+      <BpkIcon
+        style={styles.singleIcon}
+        icon="flight"
+      />
+      <BpkIcon
+        style={styles.singleIcon}
+        icon="cars"
+      />
+      <BpkIcon
+        style={styles.singleIcon}
+        icon="hotels"
+      />
+    </View>
+  </View>);
+}
+
+function getColouredIcons() {
```

Comment:
Any particular reason we aren't using an arrow function here?

Reply:
none, i didn't felt the need for it! it'll be more consistent with the rest of the file tough so I'll update, thanks